app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCPane.scala

```diff
@@ -52,48 +44,66 @@ class DLCPane(glassPane: VBox) {
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onOffer()
     }
+    tooltip = Tooltip(
+      "Initiates a DLC with the given oracle and contract parameters, generating an Offer message.")
+    tooltip.value.setShowDelay(new javafx.util.Duration(100))
   }
 
   private val acceptButton = new Button {
     text = "Accept"
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onAccept()
     }
+    tooltip = Tooltip("In response to an Offer, generates an Accept message.")
```

Comment:
"Allows you to paste in an offer and decode it. If the parameters are desirable, you can generate an Accept Message"

Reply:
This is too long, I want Peter to be able to hover over each of them and quickly see that the order is

Offer -> Accept -> Sign -> Add Sigs -> Get Funding

Reply:
Yeah, I think the purpose of the tool tip is to answer the question "What the heck is an offer?" Whereas I think you assume they already know what an offer is, and are strictly concerned about the next message.

What question are you trying to answer with the tool tips? I think we might be trying to solve different problems. We should probably resolve that difference before continuing forward cc @benthecarman